public/stylesheets/site/2.0/05-region-main.css

```diff
@@ -43,6 +43,7 @@ http://otwcode.github.com/docs/front_end_coding/patterns/supertypes
 
 #main.session {
   background: url("/images/logo.png") top left no-repeat;
+  background-position: 0.5em 3em;
```

Comment:
I saw your comment on the issue:
>I also used the background-position property as the dimensions weren’t recognised as part of the background property.

Is that true even if you remove the current position value (`top left`) from the `background` properties? `background: url("/images/logo.png") 0.5em 3em no-repeat` seems to work correctly in my browser (Safari).

Reply:
Ohh, I can't check but I suspect I forgot to remove the values, d'oh! :) `background: url("/images/logo.png") 0.5em 3em no-repeat` wfm; will update the patch.